Islandora/documentation
# Update docs_style_guide.md

**GitHub Issue**: #1040 
* Other Relevant Links (Google Groups discussion, related pull requests, Release pull requests, etc.)

# What does this Pull Request do?

Fixes the tip about how to make Tips so that it should actually render properly in the docs Right now, [it's not](https://islandora.github.io/documentation/contributing/docs_style_guide/).

# What's new?
I think it needs a title to work properly.

# How should this be tested?

run mkdocs on the docs and see if the tip presents as a nice blue box, as seen [here](https://islandora.github.io/documentation/user-documentation/media/)

# Interested parties
@Islandora/8-x-committers
